src/app/views/settings/apps/apps.html

```diff
@@ -9,12 +9,12 @@
               <div class="row apps-box">
                 <div class="app-wrapper col-xs-3 col-sm-2 col-md-2" ng-repeat="app in vm.enabledApps">
                   <div class="small-app-card ellipsis">
-                    <i class="fa fa-trash app-remove" ng-if="vm.tenantManagement" ng-click="vm.openRemoveAppModal(app, $index)" uib-tooltip="{{'mnoe_admin_panel.dashboard.organization.remove_app' | translate:{'app_name': app.name} }}" tooltip-append-to-body="true"></i>
+                    <i class="fa fa-trash app-remove" ng-if="vm.tenantManagement" ng-hide="vm.isAccountManager" ng-click="vm.openRemoveAppModal(app, $index)" uib-tooltip="{{'mnoe_admin_panel.dashboard.organization.remove_app' | translate:{'app_name': app.name} }}" tooltip-append-to-body="true"></i>
```

Comment:
Could we bundle the conditions together? I don't like mixing `ng-if` and `ng-hide`.
Might have to define a method in the controller to keep it cleaner